extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -264,140 +262,137 @@ public function getInTrackStatic() {
 	}
 
 	public function getInTrackEditPlusCategory() {
-		if($this->fetchAll()) {
+		if( $this->fetchAll() ) {
 			return $this->mInTrackEditPlusCategory;
-		}
-		else
+		} else
 			return null;
 	}
 
 	public function getNotInTrackCommunityPlatinum() {
-		if($this->fetchAll()) {
+		if( $this->fetchAll() ) {
 			return $this->mNotInTrackCommunityPlatinum;
-		}
-		else
+		} else
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)